Sources/SwiftDocC/Infrastructure/Symbol Graph/SymbolGraphLoader.swift

```diff
@@ -232,14 +232,24 @@ struct SymbolGraphLoader {
                         newAvailabilityItems.append(macCatalystAvailability)
                     }
 
-                    // If a symbol doesn't have any availability annotation at all
-                    // for a given platform, create a new one just with the
-                    // introduced version so that it shows up in the sidebar.
+                    // If a symbol doesn't have any availability version
+                    // annotation at all for a given platform, create a new
+                    // one just with the introduced version so that it shows
+                    // up in the sidebar.
                     for defaultAvailability in defaultAvailabilities {
```

Comment:
would this code be easier to read if we loop over newAvailabilityItems instead of defaultAvailabilities? and refactor what the loop does?